Sources/PrimerSDK/Classes/Data Models/Apaya.swift

```diff
@@ -82,4 +82,91 @@ public struct Apaya {
             )
         }
     }
+    
+    class ViewModel {
+        
+        var carrier: Apaya.Carrier
+        var hashedIdentifier: String?
+        
+        init?(paymentMethod: PaymentMethodToken) {
+            guard paymentMethod.paymentInstrumentType == .apayaToken else { return nil }
+            guard let mcc = paymentMethod.paymentInstrumentData?.mcc,
+                  let mnc = paymentMethod.paymentInstrumentData?.mnc,
+                  let carrier = Apaya.Carrier(mcc: mcc, mnc: mnc)
+            else { return nil }
+            
+            self.carrier = carrier
+            self.hashedIdentifier = paymentMethod.paymentInstrumentData?.hashedIdentifier
+        }
+        
+    }
+
+    enum Carrier: String, Codable {
```

Comment:
do we need unit tests for these?

Reply:
Originally this was supposed to be only in the Example app's side. However, I added it on the SDK as well cause we might need it. I can add some tests.